.travis.yml

```diff
@@ -181,18 +187,38 @@ jobs:
       - bin/release_docker $TRAVIS_TAG
       - if [[ $TRAVIS_TAG != *-rc* ]] && [[ $TRAVIS_TAG != *-dev ]]; then bin/release_docker latest; fi
       name: "Pushing release to hub.docker.com"
+
     - script:
       - bin/s3 sync s3://travis-$TRAVIS_BUILD_NUMBER/build-artifacts build/package
       - openssl aes-256-cbc -K $encrypted_2698a83519b3_key -iv $encrypted_2698a83519b3_iv -in bin/package/android/key.gpg.enc -out bin/package/android/key.gpg -d
       - gpg --import bin/package/android/key.gpg
-      - gpg --list-keys
       - bin/release_android "$TRAVIS_TAG"
       name: "Publish Android SDK to the Sonatype and Maven Central"
+
     - script:
       - bin/travis_scripts/trigger_desktop_app_build
       name: "Trigger Desktop APP build"
       if: tag = env(BUILD_DEV_RELEASE)
 
+    - script:
+      - openssl aes-256-cbc -K $encrypted_2698a83519b3_key -iv $encrypted_2698a83519b3_iv -in bin/package/android/key.gpg.enc -out bin/package/android/key.gpg -d
+      - gpg --import bin/package/android/key.gpg
+      - openssl aes-256-cbc -K $encrypted_2c668321e34e_key -iv $encrypted_2c668321e34e_iv -in bin/package/ppa/ppa_key.enc -out ~/.ssh/id_rsa -d
+      - chmod 600 ~/.ssh/id_rsa
+      - bin/release_ppa "$PPA_VERSION" "$BUILD_NUMBER" xenial
+      - bin/release_ppa "$PPA_VERSION" "$BUILD_NUMBER" bionic
+      name: "Publish DEB packages to the PPA"
+      addons:
+        apt:
+          packages:
+          - devscripts
```

Comment:
Did You consider to add dependencies to `builder` container and build inside it?

Reply:
Yeah, but it will make a configuration even more complicated since we need to pass two private keys into the builder. And with a builder container, it takes longer to finish a build. 
I think this way it's more clear what we are doing in this build.

Reply:
Builder mounts all project dir as inside volume. Also variables goes to there.
I am bringing this in, because @tadaskay soon will run all jobs in Docker runner, which is based on this builder

Reply:
I have registered issue for that https://github.com/mysteriumnetwork/node/issues/1086
Let's change it a bit later, and now we will have builds like this.